recipes/snpeff/4.5covid19/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set snpeff_ver = "v4_5covid19" %}
+# NOTE: if the version contains a trailing letter, use the <d>.<d>.1<l> format
+{% set version = "4.5covid19" %}
+{% set sha256 = "d4978b4ed51edba8458c8822bc6271794ef1f36b467d3942dbb56c7d5ae494ea" %}
+
+about:
+  home: 'http://snpeff.sourceforge.net/'
+  license: "LGPLv3"
+  summary: "Genetic variant annotation and effect prediction toolbox"
+
+package:
+  name: snpeff
```

Comment:
if we just change the version, why not changing then the original recipe?

Reply:
We can, I suppose, though I assume there is a reason the author does not want this release to be the official one as it has been out for three weeks but it is not (yet) the main release on [GitHub](https://github.com/pcingola/SnpEff/releases) or [SourceForge](http://snpeff.sourceforge.net/).

Reply:
mh, ok strange